cmake/utils.cmake

```diff
@@ -38,7 +38,7 @@ macro(set_source_files_compile_flags)
 endmacro(set_source_files_compile_flags)
 
 # A helper function to compile *.lua source into *.lua.c sources
-function(lua_source varname filename)
+function(lua_source varname filename symbolname)
```

Comment:
I would set '${module}_lua' as default value for '${symbolname}' to avoid
extra changes.